app/helpers/admin/breweries_helper.rb

```diff
@@ -0,0 +1,9 @@
+module Admin::BreweriesHelper
+  def brewery_medium_logo(brewery)
+    image_path = brewery.image.try(:url) ? brewery.image.medium_thumb.url : 'logo/logo-medium.png'
```

Comment:
__[rubocop]__
* Useless assignment to variable - `image_path`. [:ref](https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)
* Line is too long. [98/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)